apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -4174,7 +4179,13 @@ sc_ws_nameservice_contract_(Owner, GetVolley, ConnPid1, ConnPid2,
             QName = AddQuotes(Name0),
             QKey = AddQuotes(Key0),
             Args = <<"(", QName/binary, ",", QKey/binary,")">>,
-            Tx = call_a_contract(<<"can_resolve">>,
+            Fun = <<"can_resolve">>,
```

Comment:
For a variable name `Fun`, I expect to see a `fun()` type. :)
Perhaps something like `Cmd` or `Op` instead?

Reply:
You're right. Since this is the smart contract's function to be called, I will rename it to `FunName`